pkg/plugin/trivy/plugin.go

```diff
@@ -1318,6 +1346,25 @@ func (p *plugin) toSummary(vulnerabilities []v1alpha1.Vulnerability) v1alpha1.Vu
 	return vs
 }
 
+func (p *plugin) secretSummary(secrets []v1alpha1.ExposedSecret) v1alpha1.ExposedSecretSummary {
+	var s v1alpha1.ExposedSecretSummary
+	for _, v := range secrets {
+		switch v.Severity {
+		case v1alpha1.SeverityCritical:
+			s.CriticalCount++
+		case v1alpha1.SeverityHigh:
+			s.HighCount++
+		case v1alpha1.SeverityMedium:
+			s.MediumCount++
+		case v1alpha1.SeverityLow:
+			s.LowCount++
+		default:
+			s.UnknownCount++
```

Comment:
Not sure if severity `unknown` is relevant to secret world. its more related to vulnerability scanning when it comes from an external DB and its, in secret we define the severity so unknown not likely to happen